src/sources/AM32/eeprom.js

```diff
@@ -130,310 +130,17 @@ const AM32_LAYOUT = {
-    step: 1,
-  },
-  {
-    name: 'INTERVAL_TELEMETRY',
-    type: 'bool',
-    label: 'escIntervalTelemetry',
-  },
-  {
-    name: 'SERVO_LOW_THRESHOLD',
-    type: 'number',
-    label: 'escServoLowThreshold',
-    min: 750,
-    max: 1250,
-    step: 1,
-    displayFactor: 2,
-    displayOffset: 750,
-  },
-  {
-    name: 'SERVO_HIGH_THRESHOLD',
-    type: 'number',
-    label: 'escServoHighThreshold',
-    min: 1750,
-    max: 2250,
-    step: 1,
-    displayFactor: 2,
-    displayOffset: 1750,
-  },
-  {
-    name: 'SERVO_NEUTRAL',
-    type: 'number',
-    label: 'escServoNeutral',
-    min: 1374,
-    max: 1630,
-    step: 1,
-    displayFactor: 1,
-    displayOffset: 1374,
-  },
-  {
-    name: 'SERVO_DEAD_BAND',
-    type: 'number',
-    label: 'escServoDeadBand',
-    min: 0,
-    max: 100,
-    step: 1,
-  },
-  {
-    name: 'LOW_VOLTAGE_CUTOFF',
-    type: 'bool',
-    label: 'escLowVoltageCutoff',
-  },
-  {
-    name: 'LOW_VOLTAGE_THRESHOLD',
-    type: 'number',
-    label: 'escLowVoltageThreshold',
-    min: 250,
-    max: 350,
-    step: 1,
-    displayFactor: 1,
-    displayOffset: 250,
-  },
-  {
-    name: 'RC_CAR_REVERSING',
-    type: 'bool',
-    label: 'escRcCarReversing',
-  },
-];
-
-const AM32_SETTINGS_DESCRIPTIONS = {
-  '0': { base: AM32_SETTINGS_LAYOUT_0 },
-  '1': { base: AM32_SETTINGS_LAYOUT_1 },
-};
-
-const AM32_INDIVIDUAL_SETTINGS = [
-  {
-    name: 'MOTOR_DIRECTION',
-    type: 'bool',
-    label: 'escDirectionReversed',
-  },
-  {
-    name: 'BIDIRECTIONAL_MODE',
-    type: 'bool',
-    label: 'escBidirectionalMode',
-  },
-];
-
-const AM32_INDIVIDUAL_SETTINGS_DESCRIPTIONS = {
-  '0': { base: AM32_INDIVIDUAL_SETTINGS },
-  '1': { base: AM32_INDIVIDUAL_SETTINGS },
-};
-
-const AM32_DEFAULTS = {
-  '0': {
-    MOTOR_DIRECTION: 0,
-    BIDIRECTIONAL_MODE: 0,
-    SINUSOIDAL_STARTUP: 0,
-    COMPLEMENTARY_PWM: 1,
-    VARIABLE_PWM_FREQUENCY: 1,
-    STUCK_ROTOR_PROTECTION: 1,
-    TIMING_ADVANCE: 2,
-    PWM_FREQUENCY: 24,
-    STARTUP_POWER: 100,
-    MOTOR_KV: 55,
-    MOTOR_POLES: 14,
-    BRAKE_ON_STOP: 1,
-    STALL_PROTECTION: 0,
-  },
-  '1': {
-    MOTOR_DIRECTION: 0,
-    BIDIRECTIONAL_MODE: 0,
-    SINUSOIDAL_STARTUP: 0,
-    COMPLEMENTARY_PWM: 1,
-    VARIABLE_PWM_FREQUENCY: 1,
-    STUCK_ROTOR_PROTECTION: 1,
-    TIMING_ADVANCE: 2,
-    PWM_FREQUENCY: 24,
-    STARTUP_POWER: 100,
-    MOTOR_KV: 55,
-    MOTOR_POLES: 14,
-    BRAKE_ON_STOP: 1,
-    STALL_PROTECTION: 0,
-    BEEP_VOLUME: 5,
-    INTERVAL_TELEMETRY: 0,
-    SERVO_LOW_THRESHOLD: 128,
-    SERVO_HIGH_THRESHOLD: 128,
-    SERVO_NEUTRAL: 128,
-    SERVO_DEAD_BAND: 50,
-    LOW_VOLTAGE_CUTOFF: 0,
-    LOW_VOLTAGE_THRESHOLD: 50,
-    RC_CAR_REVERSING: 0,
-  },
-};
-
 const EEPROM = {
-  DEFAULTS: AM32_DEFAULTS,
-  EEPROM_OFFSET: AM32_EEPROM_OFFSET,
-  INDIVIDUAL_SETTINGS_DESCRIPTIONS: AM32_INDIVIDUAL_SETTINGS_DESCRIPTIONS,
-  LAYOUT: AM32_LAYOUT,
-  LAYOUT_SIZE: AM32_LAYOUT_SIZE,
+  EEPROM_OFFSET: EEPROM_OFFSET,
+  LAYOUT: LAYOUT,
```

Comment:
`LAYOUT,` is enough - generally if the property has the same name as the variable you can ommit it. Technically all the variables above could be renamed so that this object simply becomes:

```javascript

const EEPROM = {
  EEPROM_OFFSET,
  LAYOUT,
  LAYOUT_SIZE,
  NAMES: [''],
  PAGE_SIZE,
  RESET_DELAY,
  VERSION_OFFSET,
  VERSION_SIZE,
  BOOT_LOADER_OFFSET,
  BOOT_LOADER_SIZE,
  BOOT_LOADER_PINS,
};
```